src/Engine.zig

```diff
@@ -53,10 +53,28 @@ pub fn setOptions(engine: *Engine, options: structs.Options) !void {
     engine.options = options;
 }
 
+// Signals mach to stop the update loop.
 pub fn setShouldClose(engine: *Engine, value: bool) void {
     engine.internal.setShouldClose(value);
 }
 
+// Signals mach to wait for an event with timeout before calling update()
+// again.  Mach resets to null, so call during each update() if needed.
```

Comment:
Why are we resetting it every frame? My general understanding states that setXyz() functions are meant to do permanent changes.

Reply:
Hadn't thought of making the change permanent.  I think most people will either not call it at all or call it every frame.  I can make that change.